shell/browser/electron_browser_main_parts_linux.cc

```diff
@@ -115,17 +118,21 @@ std::string MaybeFixPlatformName(const std::string& ozone_platform_hint) {
 }  // namespace
 
 void ElectronBrowserMainParts::DetectOzonePlatform() {
+  auto env = base::Environment::Create();
```

Comment:
_(nitpick)_ this can be `const`

Reply:
Fixed in latest commit (although I'm not sure if `auto const` or `const auto` is preferred).